src/components/Menu/index.jsx

```diff
@@ -9,6 +10,7 @@ const Menu = ({ homeRef, aboutRef, contactRef, skillsRef, projectsRef }) => {
       <MenuItem title="Skills" reference={skillsRef} />
```

Comment:
Try to remove hardcode titles from component. Make it as reusable as possible

Reply:
Should I make an array of { title, ref } objects in the component and map them to create a list of <MenuItem/> components?